src/main/resources/META-INF/plugin.xml

```diff
@@ -64,6 +64,7 @@
 
     <change-notes><![CDATA[
       <ul>
+        <li>10.2 - Add support for Gradle Kotlin DSL. Bug fixes, fewer FPs and improvements for many languages.</li>
```

Comment:
We could mention S6785 from Python here.

Reply:
Why do you think so? S6785 is not new, it's an improvement.

Reply:
I added the mention because it's new, thanks